web-registry/src/types/ledger/ecocredit.ts

```diff
@@ -2,37 +2,52 @@ import type { PageResponse } from './base';
 import type { QueryBalanceResponse as BankQueryBalanceResponse } from '@regen-network/api/lib/generated/cosmos/bank/v1beta1/query';
 
 // Response structure based on https://buf.build/regen/regen-ledger
-export interface BatchRowData {
+export interface BatchData {
```

Comment:
let's be consistent with regen-ledger naming
this will also ease the migration to regen-js for queries
```suggestion
export interface BatchInfo {
```